### ISSUE TITLE
Improve Palette Picker

### ISSUE THREAD

**Issue by oceanhahn** [aseprite/aseprite#2365]
Just a couple of simple-ish UI things I that I think would make the Palette Picker more robust.

![image](https://user-images.githubusercontent.com/11544843/81008678-520efb80-8e21-11ea-8d4a-f862b876328b.png)
 🔼 _This bad boy._

 - **Palette Groups:** Lets users sort their palettes into named categories by dragging and dropping palettes into collapsable/expandable folders, so they do not need to see every one of them every time. May or may not allow one palette to appear in multiple groups; that is neither here nor there for me.

 - **Recent Palettes List:** It would be helpful if there was a list of recently-loaded or saved palettes, in the same way that _Open Recent_ does for saved and loaded image files.
Also consider adding a "Add to Palette Picker" option to the Load pane (when loading palettes) that would list the loaded palette if it is not already there, maybe!

Thanks for considering it! :D 

..using ASEprite 1.2.18-x86 on Win10.

**Comment by ccioanca** [comment]
Palettes should definitely allow for "grouping". It would make it a lot simpler for artists to set specific colors for specific elements for their drawing/animation. Especially when working with shades.

**Comment by dacap** [comment]
Actually here I think @oceanhahn is talking about grouping preset palettes, and you @ccioanca are looking for something more like https://community.aseprite.org/t/allow-to-group-colors-in-a-palette/445  or https://community.aseprite.org/t/group-swatches-within-palette/372/4?u=dacap

**Comment by ccioanca** [comment]
@dacap you're totally right. Although I agree with both enhancements. Is there a git issue for the above links or do you prefer suggestions to go to `community.aseprite.org`?

**Comment by dacap** [comment]
I prefer github but the community site is fine too (sometimes we create issues from the community, or refer to the community directly). I've found a github issue related to the color groups too: #1575

**Comment by oceanhahn** [comment]
@dacap has it right: this is about grouping multiple palettes, not grouping colours within a single palette.

I think I posted an issue about colour grouping somewhere else, or at least read one on Github.

It may have been per-frame palettization for .gifs.
As well as a lot of other stuff about .gifs. <_<;

**Comment by dacap** [comment]
The palette popup needs some love, some quick features are showing the current palette as first item and default as second and then the rest of the list: https://community.aseprite.org/t/current-palette-in-palette-list/22155

Probably we should be able to pin palettes just like the recent list of files.
